app/views/accounts/_auth_apps.html.erb

```diff
@@ -9,9 +9,11 @@
           <%= auth_app_configuration.name %>
         </div>
       </div>
-      <div class="grid-col-4 text-right">
-        <%= render 'accounts/actions/disable_totp', id: auth_app_configuration.id %>
-      </div>
+      <% if MfaPolicy.new(current_user).multiple_non_restricted_factors_enabled? %>
```

Comment:
Just to clarify for my understanding: I'm assuming we should have had a condition here previously for `multiple_factors_enabled?`, as we did in _some_-but-not-all of the MFA partials (e.g. backup codes), and we're introducing it now for the non-feature-flagged path for consistency?

Reply:
Yes, this seems like it was an oversight in the past since we want to prevent someone from deleting their last mfa method.